### DISCUSSION TITLE
[Request] flux-pro release notes (anystack changelog)

### DISCUSSION THREAD

**Discussion by idealerror** [livewire/flux#138] [Feature Requests] [answered]
Could the public changelog for `flux-pro` be enabled within anystack?  This way developers will be able to see the release notes. 

Example: https://changelog.anystack.sh/observer

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

**Comment by joshhanley** [comment, marked as answer]
Going to close this as we've enabled the Anystack change log, but also the Flux GitHub repo releases reflects the same information.

https://changelog.anystack.sh/flux-pro
https://github.com/livewire/flux/releases
